sdk/maps/maps-geolocation-rest/README.md

```diff
@@ -0,0 +1,138 @@
+# Azure Maps Geolocation REST client library for JavaScript
+
+Azure Maps Geolocation Client
+
+\*\*If you are not familiar with our REST client, please spend 5 minutes to take a look at our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library, the REST client provides a light-weighted & developer friendly way to call azure rest api
+
+Key links:
+
+- [Source code][source_code]
+- [Package (NPM)][npm_package]
+- [API reference documentation][api_ref]
+- [Samples][samples]
+- [Product Information][product_info]
+
+## Getting started
+
+### Currently supported environments
+
+- [LTS versions of Node.js](https://nodejs.org/about/releases/)
+- Latest versions of Safari, Chrome, Edge and Firefox.
+
+### Prerequisites
+
+- You must have an [Azure subscription][az_subscription] to use this package.
+- An [Azure Maps account][az_maps_account_management]. You can create the resource via the [Azure Portal][azure_portal], the [Azure PowerShell][azure_powershell], or the [Azure CLI][azure_cli].
+
+If you use Azure CLI, replace `<resource-group-name>` and `<map-account-name>` of your choice, and select a proper [pricing tier][az_map_pricing] based on your needs via the `<sku-name>` parameter. Please refer to [Azure Maps Reference for Azure CLI][az_map_az_cli] for more details.
+
+```bash
+az maps account create --resource-group <resource-group-name> --name <map-account-name> --sku <sku-name>
+```
+
+### Install the `@azure-rest/maps-geolocation` package
+
+Install the Azure MapsGeolocation REST client REST client library for JavaScript with `npm`:
```

Comment:
```suggestion
Install the Azure Maps Geolocation REST client REST client library for JavaScript with `npm`:
```

Reply:
Updated in [eb12dac](https://github.com/Azure/azure-sdk-for-js/pull/23956/commits/eb12dac47f44dbf4ffa3df8ab627bea3d4d2ce1d)